### ISSUE TITLE
Brakeman: Malformed Host header

### ISSUE THREAD

**Issue by milgner** [codeclimate/codeclimate#459]
Trying to run `analyze` on my machine immediately reports the error:

```
Starting analysis
Running brakeman: Done!
error: (CC::Analyzer::Engine::EngineFailure) engine brakeman:stable failed with status 1 and stderr 
time="2016-06-14T08:53:10Z" level=fatal msg="Error response from daemon: 400 Bad Request: malformed Host header" 
```

I don't know whether this is specific to brakeman and since I couldn't find an engine repository for it, I'm posting it here.

Other commands like `help` seem to work and `validate-config` reports `No errors or warnings found in .codeclimate.yml file.`

I'm running `Docker version 1.11.1, build 5604cbe`.
`uname -a`: `Linux nandus.alveran 4.5.4-1-ARCH #1 SMP PREEMPT Wed May 11 22:21:28 CEST 2016 x86_64 GNU/Linux`
`env | grep "^DOCKER_"`: - nothing -

Let me know if I can provide any additional information.

**Comment by pbrisbin** [comment]
Hi, are you a fellow Arch Linux user?

I ran into this because Arch's docker binary is built with go-1.6 (latest) but the client installed in the CLI is built with go-1.5 (as upstream intends). This makes them incompatible. I opened a bug about this [here](https://bugs.archlinux.org/task/49207) with more details.

To confirm this issue, take a look at `docker version`, it'll tell you which version of go was used to build the binary -- if it's 1.6, that's it.

As a workaround, I built and installed my own docker package against go-1.5. If you are in fact on Arch, I can give you more specific instructions, it's very easy to do on that distro.
